### ISSUE TITLE
allow us to see what thing carries what.

### ISSUE THREAD

**Issue by SoggyLemon** [cataclysmbnteam/Cataclysm-BN#1546]
### Is your feature request related to a problem? Please describe.

its hard telling whats in what pocket because theres no way of telling.

### Describe the solution you'd like

let players see what pocket is carrying what. for example highlight the items my backpack is carrying when i hover over my backpack in the inventory window.

**Comment by Proxiehunter** [comment]
Things aren't in a specific pocket in Bright Nights. That's how they avoid the over complicated mess that led to me switching to Bright Nights instead of no longer playing DDA at all.

**Comment by SoggyLemon** [comment]
> 

yeah i dont mind that, i just want to see what holds what, as sometimes i drop clothing i didnt know had important stuff in them.

**Comment by SoggyLemon** [comment]
> Things aren't in a specific pocket in Bright Nights. That's how they avoid the over complicated mess that led to me switching to Bright Nights instead of no longer playing DDA at all.

yeah i dont mind that, i just want to see what holds what as sometimes i drop clothing that i didnt know had important stuff in it

**Comment by Proxiehunter** [comment]
The way I understand the system to currently work is that your clothing _doesn't_ "have stuff in it". You drop your clothing and it runs a quick check of what it can drop to get you down to the correct volume.

I was _going_ to say it wouldn't work without introducing the over complication of the pockets system but . . .

Could it run that check and display the results somewhere of what _would_ be dropped if you dropped a specific piece of clothing? A problem I foresee with that is that it has to recalculate every single time you pick something up or put on/take off a bit of clothing. That might slow things down too much. But theoretically that _could_ be done couldn't it?

Maybe run the calculation and tell you what you would drop and give you a Y/N prompt at the time you drop or take off the clothing. That would keep how many times the calculation has to be run to the same number. 

If any of that means it would be easier than I thought then it might be worth actually doing.

**Comment by scarf005** [comment]
> I was _going_ to say it wouldn't work without introducing the over complication of the pockets system but . . .
> 
> Could it run that check and display the results somewhere of what _would_ be dropped if you dropped a specific piece of clothing? A problem I foresee with that is that it has to recalculate every single time you pick something up or put on/take off a bit of clothing. That might slow things down too much. But theoretically that _could_ be done couldn't it?


#603 does a groundwork for this exact feature. it's just... we don't have an *actual* means to mark them, so 'dumb' drops for now.

**Comment by olanti-p** [comment]
Related to https://github.com/cataclysmbnteam/Cataclysm-BN/issues/1546
